doc/experiment.rst

```diff
@@ -12,316 +38,231 @@ Jupyter notebook, interactive python session, or Python file and use the
 ``Experiment`` to iteratively create, configure and launch computational kernels
 on the system through the specified launcher.
 
-.. |SmartSim Architecture| image:: images/ss-arch-overview.png
-  :width: 700
-  :alt: Alternative text
-
-|SmartSim Architecture|
-
-
 The interface was designed to be simple, with as little complexity as possible,
 and agnostic to the backend launching mechanism (local, Slurm, PBSPro, etc.).
 
-Model
-=====
-
-``Model(s)`` are subclasses of ``SmartSimEntity(s)`` and are created through the
-Experiment API. Models represent any computational kernel. Models are flexible
-enough to support many different applications, however, to be used with our
-clients (SmartRedis) the application will have to be written in Python, C, C++,
-or Fortran.
-
-Models are given :ref:`RunSettings <rs-api>` objects that specify how a kernel
-should be executed with regard to the workload manager (e.g. Slurm) and the
-available compute resources on the system.
-
-Each launcher supports specific types of ``RunSettings``.
-
-   - :ref:`SrunSettings <srun_api>` for Slurm
-   - :ref:`AprunSettings <aprun_api>` for PBSPro and Cobalt
-   - :ref:`MpirunSettings <openmpi_run_api>` for OpenMPI with `mpirun` on PBSPro, Cobalt, LSF, and Slurm
-   - :ref:`JsrunSettings <jsrun_api>` for LSF
-
-These settings can be manually specified by the user, or auto-detected by the
-SmartSim Experiment through the ``Experiment.create_run_settings`` method.
-
-A simple example of using the Experiment API to create a model and run it
-locally:
-
-.. code-block:: Python
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime.
 
-  from smartsim import Experiment
+==========
+ Launchers
+==========
 
-  exp = Experiment("simple", launcher="local")
+The Experiment API *interfaces* both locally and with four
+different job schedulers that SmartSim refers to as `launchers`. `Launchers`
+provide SmartSim the ability to construct and execute complex workloads
+on HPC systems with schedulers (workload managers) like Slurm, or PBS.
 
-  settings = exp.create_run_settings("echo", exe_args="Hello World")
-  model = exp.create_model("hello_world", settings)
+SmartSim currently supports 5 `launchers`:
+  1. ``local``: for single-node, workstation, or laptop
+  2. ``slurm``: for systems using the Slurm scheduler
+  3. ``pbs``: for systems using the PBSpro scheduler
+  4. ``cobalt``: for systems using the Cobalt scheduler
+  5. ``lsf``: for systems using the LSF scheduler
+  6. ``auto``: have SmartSim auto-detect the launcher to use.
```

Comment:
Standardize punctuation on list items - I have no preference but would like to see consistency across list items and lists